requirements/pytorch/base.txt

```diff
@@ -1,10 +1,10 @@
 numpy>=1.17.2, <1.23.1
 torch>=1.9.*, <=1.12.0
-tqdm>=4.57.0, <=4.63.0
+tqdm>=4.57.0, <4.65.0
 PyYAML>=5.4, <=6.0
 fsspec[http]>=2021.05.0, !=2021.06.0, <2022.6.0
 tensorboard>=2.9.1, <2.10.0
-torchmetrics>=0.7.0, <0.9.2  # needed for using fixed compare_version
+torchmetrics>=0.7.0, <0.9.3  # needed for using fixed compare_version
```

Comment:
I guess we can still merge this, but I'm completely unsure why the bot updates `torchmetrics' version as well...
```suggestion
torchmetrics>=0.7.0, <0.9.2  # needed for using fixed compare_version
```